jsapp/js/components/common/button.tsx

```diff
@@ -10,20 +10,18 @@ import cx from 'classnames';
 
 /**
  * Note: we use a simple TypeScript types here instead of enums, so we don't
- * need to import them, just pass correct strings.
+ * need to import them - just pass correct strings.
  */
 
 /**
  * Button types are:
- * 1. bare - no border, no background, hover sets background
- * 2. frame - border, no background, hover sets background
- * 3. full - no border, background, hover dims background
+ * 1. primary - white text on blue background
+ * 2. secondary - dark blue text on light blue background
+ * 3. danger - white text on red background
+ * 4. secondary danger - dark red text on light red background
+ * 5. text - dark blue text with no background
  */
-export type ButtonType = 'bare' | 'frame' | 'full';
-export type ButtonColor =
-  | 'blue'
-  | 'red'
-  | 'dark-blue';
+export type ButtonType = 'primary' | 'secondary' | 'danger' | 'secondary-danger' | 'text';
```

Comment:
Tiny nitpick. Is there a way to add a jsdoc to this type such that it shows up when auto-completing? At first glance these names are a bit vague and requires the user to read this component whereas before it was a bit more descriptive. 
![image](https://github.com/user-attachments/assets/754a85bf-5d87-42da-9f08-a690d54d03f2)

Reply:
![image](https://github.com/user-attachments/assets/41482b08-da68-45ed-8649-141d07ab9219)
(Just checking it is the same on vscode also)

Reply:
There is a JSDoc comment for the line where type is defined, but I will move it to the line where button prop is - it should do the trick :ok: